samples/server/petstore/php-slim/lib/Api/AbstractAnotherFakeApi.php

```diff
@@ -73,7 +73,5 @@ public function call123TestSpecialTags(ServerRequestInterface $request, Response
         $body = $request->getParsedBody();
         $message = "How about implementing call123TestSpecialTags as a PATCH method in OpenAPIServer\Api\AnotherFakeApi class?";
         throw new Exception($message);
```

Comment:
Declaration of $message variable is useless if only used once. I would paste the message to the constructor directly